django_project/utils/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+
+from . import views
+
+urlpatterns = [
+    path('', views.renderHTML, name='renderHTML'),
+    path('<subpath>/', views.renderHTML, name='renderHTML'),
```

Comment:
When you go to `http://localhost:8000/complex/path/thing`, it will not match either of these paths